backend/inspirehep/records/api/literature.py

```diff
@@ -507,6 +508,9 @@ def add_file(
             new_key = hash_data(file_data)
             mimetype = magic.from_buffer(file_data, mime=True)
             file_data = BytesIO(file_data)
+            is_scanned = None
+            if "fulltext" not in file_data and mimetype == "application/pdf":
```

Comment:
`file_data` is `BytesIO` why checking if `fulltext` is not there?

Reply:
`fulltext` is a mark and we wanted to avoid double-checking if they're scanned for the documents that we already checked.